src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -117,23 +88,26 @@ private void initializeCommandTable() {
                         "Automatic",
                         "No command required"),
```

Comment:
Should we swap the order of commands by its importance, reflecting what we order in the UG

Reply:
Here's the most recent version (the status haven't been added to the UG's one yet tho)
<img width="795" alt="Screenshot 2567-11-04 at 17 31 37" src="https://github.com/user-attachments/assets/effbb28a-ba87-4635-b777-70351d78c25a">

Reply:
Yup, I think we should follow this (or even consider sorting the commands based on whether they modify data or are purely for viewing).

Saving data should be moved to the bottom as well since users are unlikely to reference it often.

Reply:
👍